base/applications/regedit/find.c

```diff
@@ -223,16 +231,19 @@ BOOL RegFindRecurse(
                                   NULL, &cb);
         if (lResult != ERROR_SUCCESS)
             goto err;
-        pb = malloc(cb);
+        pb = malloc(cb + 3); /* To avoid buffer overrun, append 3 NULs */
```

Comment:
You should only need to add two (2) more bytes so that you can put a UNICODE NULL at the end.

Reply:
`cb` can be an odd number. 2 bytes is not enough.

Reply:
If cb is an odd number, there is probably something wrong. Maybe ASSERT this after the type check?